Assets/UI/Panels/productionpanel_CQUI.lua

```diff
@@ -214,22 +215,29 @@ function GetData()
     local buildQueue = pSelectedCity:GetBuildQueue();
 
     for row in GameInfo.Units() do
-        if row.MustPurchase and buildQueue:CanProduce( row.Hash, true ) and row.PurchaseYield == "YIELD_FAITH" then
```

Comment:
It should be (row.PurchaseYield == "YIELD_FAITH" or row.PurchaseYield == "YIELD_GOLD") as these are the only yields supported atm. Less likely to get errors from mods.

Reply:
yeah, good call.  fixing.